code/modules/mob/living/carbon/xenomorph/castes/spiderling/spiderling.dm

```diff
@@ -103,7 +110,7 @@
 			change_action(ESCORTING_ATOM, escorted_atom)
 			return
 	mob_parent.face_atom(atom_to_walk_to)
-	mob_parent.UnarmedAttack(atom_to_walk_to, mob_parent)
+	addtimer(CALLBACK(mob_parent, TYPE_PROC_REF(/mob, UnarmedAttack), atom_to_walk_to, mob_parent), rand(1, 15) MILLISECONDS)
```

Comment:
This is shitcode. Why?

Reply:
Timing is randomized as all the spiderlings slashing at the same time is too robotic, deals incredible amounts of damage, so it's better that it's a constant stream of slashing instead, and it looks better too

Reply:
I told you to remove the milliseconds and you didn't, I don't know why you keep bugging me for review

Reply:
I thought I had done it, but i was mistaken as I had removed all but this one, I kept bugging you for a re-review because I assumed it was needed for a testmerge